js/scripts.js

```diff
@@ -459,4 +459,39 @@ function showPopupWindow() {
 
 showPopupWindow(); 
 
-} //end of for desktop version
\ No newline at end of file
+} //end of popup window for desktop version
+
+
+
+
```

Comment:
Kindly avoid leaving unnecessary spaces in your code to make it more readable and concise.